apps/css-workshop/src/components/Radio.astro

```diff
@@ -0,0 +1,43 @@
+---
+type Props = {
+  status?: 'positive' | 'warning' | 'negative';
+  disabled?: boolean;
+  checked?: boolean;
+  labelPosition?: 'left' | 'right';
+  value: string;
+  name: string;
+} & astroHTML.JSX.HTMLAttributes;
+
+const { disabled, status, class: className, labelPosition, ...props } = Astro.props;
+---
+
+<label
+  class:list={[
+    'iui-radio-wrapper',
+    {
+      'iui-disabled': disabled,
+      [`iui-${status}`]: !!status,
+    },
+  ]}
+>
+  {
+    labelPosition === 'left' && (
+      <>
+        <span class='iui-radio-label'>
+          <slot />
+        </span>
+      </>
+    )
+  }
+  <input class:list={['iui-radio', className]} type='radio' disabled={disabled} {...props} />
+  {
+    labelPosition === 'right' ||
+      (!labelPosition && (
```

Comment:
This if condition looks wrong. When `labelPosition` is set to `'right'`, it will *not* show this label.

Please wait for a second approval before merging the PR next time. Such mistakes would be easily caught by a second review.

Additionally please try to respond to the comments after committing code. If you don't respond, the reviewers may not even realize that you acted on their comments.

Reply:
I apologize for this and will wait for another round of review next time as well as make sure to notify every commentor about changes. I will open another PR to fix this issue. I will open another PR to fix this issue.